### PR TITLE
docs: Install Node with Homebrew

### PR DESCRIPTION
**Pull request by marcysutton**
## Description

This PR is a start to documenting Homebrew & Node install instructions as discussed in #10398. 

One thing I considered but didn't commit was a callout for more specific instructions around global modules, such as mentioned in https://github.com/gatsbyjs/gatsby/issues/10398#issuecomment-466451396.

> To troubleshoot issues with globally installed packages, visit the [extended Node.js docs](/docs/troubleshooting-global-node/).

If we feel an extended doc is necessary, I can work on adding it...but I intentionally kept this simple for now. Feedback is welcome!

### CODE REVIEW

File: docs/tutorial/part-zero/index.md [hunk 1]
```diff
@@ -25,15 +25,34 @@ Take a moment to locate and open up the command line interface (CLI) for your co
...
 
-## Install Node.js
+## Install Homebrew for Node.js
+
+To install Gatsby and Node.js, it is recommended to use [Homebrew](https://brew.sh/). A little set-up in the beginning can save you from some headaches later on!
+
+How to install or verify Homebrew on your computer:
+
+1. Open your Terminal
+1. See if Homebrew is installed by running `brew -v`. You should see "Homebrew" and a version number, if so.
```
Comment (wardpeet) [inline docs/tutorial/part-zero/index.md]: the `if so` reads a bit strange in my opinion, it looks like something is missing.
Wouldn't it be better to just omit that?

Comment (marcysutton) [reply docs/tutorial/part-zero/index.md]: Makes sense to me, I've updated the PR!

**Comment by KyleAMathews** [comment]
This PR has 584 files changed

**Comment by marcysutton** [comment]
Weird, a botched merge from master. I'll try to fix it...

**Comment by marcysutton** [comment]
This PR is fixed up now! I wonder if we should add anything about what to do if you already have Node installed, though. Do we want to add anything about upgrading or uninstalling Node that was set up in other ways?

**Comment by wardpeet** [comment]
I think it is good enough as is, unsure if many people have node installed without knowing 😛

**Comment by marcysutton** [comment]
Ok! I'm going to merge this since it was approved and it doesn't have 584 files changed anymore :)

**Pull request merged**